### PR TITLE
add PR template

### LINKED ISSUE #104: Add pull request template with checklist for submitting a tutorial
The [create tutorial](https://inbo.github.io/tutorials/create_tutorial/) is quite long. A TL;DR section should summarise the key requirements for the tutorials. It can assume that the reader knows about git, github, Rstudio, ... If they don't than they can read the full document.

### PR DESCRIPTION
**Pull request by hansvancalster**
This PR adds a pull request template for new or updated tutorials or articles.
Closes #104

### CODE REVIEW

**Review by ToonHub — APPROVED** [review]
Good template

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!--- indicate the Title for this pull request (PR) above -->
+
+
+Thank you for considering to contribute to the INBO tutorials repository.
```
Comment (ElsLommelen) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:4]: As the PR is generally added after the contribution has already been made, I would remove the 'considering to'.  It somehow gives the feeling that it is not sure yet if the contribution will be added.

```suggestion
Thank you for contributing to the INBO tutorials repository.
```

**Review by ElsLommelen — APPROVED** [review]
I haven't tested it, only looked at the content.  This content looks fine, I only have one minor comment.

**Review by florisvdh — APPROVED** [review]
A fine template @hansvancalster :+1:.

I was wondering whether referral to https://github.com/inbo/tutorials/blob/master/.github/workflows/REVIEWING.md would also be relevant to include as a markdown note at the end, considering that _after creation_ of the pull request a rendered dev-website will be created automatically in GH Actions. In that way the author can play a reviewing role already, checking that everything renders OK. Not sure if that fits the idea of the PR template.

**Comment by hansvancalster** [comment]
> A fine template @hansvancalster 👍.
> 
> I was wondering whether referral to https://github.com/inbo/tutorials/blob/master/.github/workflows/REVIEWING.md would also be relevant to include as a markdown note at the end, considering that _after creation_ of the pull request a rendered dev-website will be created automatically in GH Actions. In that way the author can play a reviewing role already, checking that everything renders OK. Not sure if that fits the idea of the PR template.

Good idea. Done in 435dd8e

**Pull request merged**